static/app/views/feedback/feedbackListPage.tsx

```diff
@@ -1,64 +1,85 @@
 import {RouteComponentProps} from 'react-router';
 import styled from '@emotion/styled';
 
+import ErrorBoundary from 'sentry/components/errorBoundary';
 import FeedbackEmptyDetails from 'sentry/components/feedback/details/feedbackEmptyDetails';
 import {FeedbackDataContext} from 'sentry/components/feedback/feedbackDataContext';
 import FeedbackFilters from 'sentry/components/feedback/feedbackFilters';
 import FeedbackItemLoader from 'sentry/components/feedback/feedbackItem/feedbackItemLoader';
 import FeedbackSearch from 'sentry/components/feedback/feedbackSearch';
 import FeedbackList from 'sentry/components/feedback/list/feedbackList';
-import useFeedbackListQueryView from 'sentry/components/feedback/useFeedbackListQueryView';
 import FullViewport from 'sentry/components/layouts/fullViewport';
 import * as Layout from 'sentry/components/layouts/thirds';
 import PageFiltersContainer from 'sentry/components/organizations/pageFilters/container';
 import SentryDocumentTitle from 'sentry/components/sentryDocumentTitle';
 import {t} from 'sentry/locale';
 import {space} from 'sentry/styles/space';
-import {FeedbackItemLoaderQueryParams} from 'sentry/utils/feedback/item/types';
-import {decodeScalar} from 'sentry/utils/queryString';
-import {useLocation} from 'sentry/utils/useLocation';
+import {decodeList, decodeScalar} from 'sentry/utils/queryString';
+import useLocationQuery from 'sentry/utils/url/useLocationQuery';
 import useOrganization from 'sentry/utils/useOrganization';
 import FluidHeight from 'sentry/views/replays/detail/layout/fluidHeight';
 
 interface Props extends RouteComponentProps<{}, {}, {}> {}
 
 export default function FeedbackListPage({}: Props) {
   const organization = useOrganization();
-  const location = useLocation<FeedbackItemLoaderQueryParams>();
 
-  const queryView = useFeedbackListQueryView({
-    queryReferrer: 'feedback_list_page',
+  const queryView = useLocationQuery({
+    fields: {
+      collapse: ['inbox'],
+      expand: [
+        'owners', // Gives us assignment
+        'stats', // Gives us `firstSeen`
+      ],
+      limit: 25,
+      queryReferrer: 'feedback_list_page',
+      shortIdLookup: 0,
+      end: decodeScalar,
+      environment: decodeList,
+      field: decodeList,
+      project: decodeList,
+      query: decodeScalar,
+      start: decodeScalar,
+      statsPeriod: decodeScalar,
+      utc: decodeScalar,
+    },
+  });
```

Comment:
This used to be inside of [useFeedbackListQueryView.tsx](https://github.com/getsentry/sentry/pull/58209/files#diff-a5ff4402dff3273791562ca1a879ea447e716ade2d9363ede52f2b0a5b0e66d5), but since most of the types are optional, the type hints from having that function didn't do much, because it could return nothing that that's alright!

So just inline it here, and this component looks cool because it's loading the list params separate from the single-item param (which is on the next line).

No more `useLocation`.